### PR TITLE
Steam Test Server - Update 26

### PR DESCRIPTION
**Pull request by Sunaries**
Work in progress. Proofread IMC's newest updates of March 25.

Proofreading:
- https://github.com/Treeofsavior/EnglishTranslation/commit/c2a6582299c9108eacc2cf2d0d338f84fb1930c9 - done
- https://github.com/Treeofsavior/EnglishTranslation/commit/abb28e7e92af1d951dfe1aefb451e5d3287a3739 - done
- https://github.com/Treeofsavior/EnglishTranslation/commit/cedd825312273502a94947da2087ebf3f444f79a - done
- https://github.com/Treeofsavior/EnglishTranslation/commit/bcc2b623e24e28e9008c765bd4896386a3b9a894 - done
- https://github.com/Treeofsavior/EnglishTranslation/commit/8122e6d56299cee07e12dcda92499e65a65b5a66 - done
- https://github.com/Treeofsavior/EnglishTranslation/commit/33526264cac826314454ed3873d3b0e44a73e832 - done

### CODE REVIEW

File: EnglishTranslation-master/ETC.tsv [hunk 1]
```diff
@@ -18265,38 +18265,38 @@ ETC_20151224_018265	Contains 10 Soul Crystals when you open it. Right-click to u
...
+ETC_20151224_018285	Applies Token benefits to all your characters in the server for 3 days. Right-click to use.{nl}[Token Benefits]{nl} - Able to learn up to 2 attributes simultaneously{nl} - Reduced Market commission fee (30% {img white_right_arrow 16 16} 10%){nl} - Able to list up to 5 items on the Market{nl} - +3 increased character movement speed{nl} - +20% EXP from hunting monsters{nl}(Stacks with the Internet Cafe benefits & EXP Tome effect){nl} - Able to use Premium Gestures
 ETC_20151224_018286	Skill Reset Potion
 ETC_20151224_018287	A very useful potion for those who just cannot get their skill build right.{nl}Use this to reset your current Skill Points.
 ETC_20151224_018288	Instanced Dungeon Reset Voucher
-ETC_20151224_018289	Resets the restrictions on the instanced dungeons and mercenary missions when used.
+ETC_20151224_018289	Use this to resets the instance entry restrictions for instanced dungeons and mercenary missions.
```
Comment (Sunaries) [inline EnglishTranslation-master/ETC.tsv:18288]: Resets -> reset

File: EnglishTranslation-master/ETC.tsv [hunk 2]
```diff
@@ -21173,7 +21173,7 @@ ETC_20160315_021172	Right-click to obtain the following items:{nl} - 200 TP{nl}
 ETC_20160315_021173	You can enter the server by purchasing DLC.
```
Comment (Sunaries) [inline EnglishTranslation-master/ETC.tsv:21173]: The server -> this server

File: EnglishTranslation-master/ETC.tsv [hunk 3]
```diff
@@ -21322,60 +21322,60 @@ ETC_20160323_021321	You've stopped digging
...
+ETC_20160323_021354	Guild member {Name} received {Item} (x{Count}) from the Guild Storage. 
+ETC_20160323_021355	You've received an item from the Guild Storage. Retrieve the item in your Lodge.
+ETC_20160323_021356	You can't use this item in other Guild Hangouts.
+ETC_20160323_021357	You can only place one bonfire.{nl}The previous bonfire has been removed.
+ETC_20160323_021358	You cannot sell this.
+ETC_20160323_021359	You are not in the condition to do this action.
 ETC_20160323_021360	0
-ETC_20160323_021361	Only {Job} {Grade} circles can purchase it.
-ETC_20160323_021362	Resurrecting after a while
+ETC_20160323_021361	Only {Job} of Circle {Grade} can purchase it.
```
Comment (Sunaries) [inline EnglishTranslation-master/ETC.tsv:21361]: Only _the_

File: EnglishTranslation-master/ITEM.tsv [hunk 4]
```diff
@@ -10327,8 +10327,8 @@ ITEM_20160310_010326	Old Backpack
 ITEM_20160310_010327	An old backpack that reacts to the magic detector. It contains several objects of unknown use...
 ITEM_20160310_010328	Agnim Coral Fragment.
```
Comment (Sunaries) [inline EnglishTranslation-master/ITEM.tsv]: Requires a check. Agnim (or Arnim)

File: EnglishTranslation-master/ITEM.tsv [hunk 5]
```diff
@@ -10343,65 +10343,65 @@ ITEM_20160323_010342	Bronza Bracelet
...
+ITEM_20160323_010369	[Equip Effect] 0.8% chance to inflict 8640 additional damage against Insect-type Targets
 ITEM_20160323_010370	Magiya Amulet
-ITEM_20160323_010371	[Effects when equipped] Increases magical amplification by 152 for 30 seconds when critital hit occurs, with 5% possibility
+ITEM_20160323_010371	[Equip Effect] 5% chance to increase magic amplification by 152 for 30 seconds upon a critical
 ITEM_20160323_010372	Vinis Amulet
-ITEM_20160323_010373	[Effects when equipped] Causes bleeding effect for 3 seconds when using slash-type attacks, with 15% possibility
+ITEM_20160323_010373	[Equip Effect] 15% chance to inflict [Bleeding] for 3 seconds when using [Slash]-type attacks
 ITEM_20160323_010374	Palemidas Amulet
-ITEM_20160323_010375	[Effects when equipped] Decreases defense by 50% for 3 seconds when using thrust-type attacks, with 5% possibility
+ITEM_20160323_010375	[Equip Effect] 5% chance to decrease defense by 50% when using [Stab]-type attacks
```
Comment (Sunaries) [inline EnglishTranslation-master/ITEM.tsv]: "For 3 seconds" is missing

File: EnglishTranslation-master/ETC.tsv [hunk 6]
```diff
@@ -21220,10 +21220,10 @@ ETC_20160323_021219	0
 ETC_20160323_021220	0
 ETC_20160323_021221	0
 ETC_20160323_021222	0
-ETC_20160323_021223	90 TP [Free Gift]
-ETC_20160323_021224	90 TP will be consumed when used.
+ETC_20160323_021223	90 TP [Event]
+ETC_20160323_021224	Use it to obtain 90 TP.
 ETC_20160323_021225	Lucky 100 Package
```
Comment (Sunaries) [inline EnglishTranslation-master/ETC.tsv:21225]: Use it to obtain ... -> Use to obtain (applies to the others as well)

File: EnglishTranslation-master/SKILL.tsv [hunk 7]
```diff
@@ -6336,21 +6336,21 @@ SKILL_20160310_006335	Applies the effects of [Swift Step] to party members. (The
...
-SKILL_20160323_006352	The Critical Rate of [Multi Shot] has been increased 1.5% per Attribute Level.
-SKILL_20160323_006353	While equipped with [Two Handed Arrow], 20% additional damages will be dealt when attacking aerial type monsters per Attribute Level.
-SKILL_20160323_006354	The Critical Rate of [Spiral Arrow] will increase by 13% per Attribute Level.
-SKILL_20160323_006355	Carnivorous: Duration Time
-SKILL_20160323_006356	The duration time of [Carnivorous] increases by 1 sec per Attribute Level.
+SKILL_20160323_006349	 - Able to learn up to 2 attributes simultaneously{nl} - Reduced Market commission fee (30% {img white_right_arrow 16 16} 10%){nl} - Able to list up to 5 items on the Market{nl} - +3 increased character movement{nl} - Increased maximum buff count{nl} - Able to use Premium Gestures{nl}
+SKILL_20160323_006350	Increases the critical chance of [Double Slash] by 7% per attribute level.
+SKILL_20160323_006351	Increases the critical chance of [Moulinet] by 12% per attribute level.
+SKILL_20160323_006352	Increases the critical chance of [Multi Shot] by 7% per attribute level.
```
Comment (Sunaries) [inline EnglishTranslation-master/SKILL.tsv]: Check percentage. Correct to: 1.5%

File: EnglishTranslation-master/UI.tsv [hunk 8]
```diff
@@ -2161,11 +2161,11 @@ UI_20160323_002160	CON : {@st41b}%s
...
-UI_20160323_002164	{@st42}Hair Color{/}
+UI_20160323_002164	{@st42}Hairstyle Color{/}
 UI_20160323_002165	{@st42}Available Titles{/}
 UI_20160323_002166	Team Info
 UI_20160323_002167	{@st66b}Change Lodge{/}
-UI_20160323_002168	{@st43}{s26}Team Level  %s{/}
+UI_20160323_002168	{@st43}{s26}Team Level %s{/}
 UI_20160323_002169	{@st66b}Team Info
 UI_20160323_002170	{#2c140c}Average Level : %s{/}
 UI_20160323_002171	{@st41b}Participate
```
Comment (Sunaries) [inline EnglishTranslation-master/UI.tsv]: -> Join

File: EnglishTranslation-master/ETC.tsv [hunk 9]
```diff
@@ -21322,60 +21322,60 @@ ETC_20160323_021321	You've stopped digging
...
 ETC_20160323_021373	TP Exchange Ticket
-ETC_20160323_021374	Guild Rewards
-ETC_20160323_021375	1st Standing Server Rewards
-ETC_20160323_021376	Free TP+{Value} until the next refresh of the weekly standings
-ETC_20160323_021377	Re-enter the automatic party matching
-ETC_20160323_021378	You are already receiving the benefits of Tokens. Do you still want to apply?{nl}If you duplicate the application{nl}The entire usage perios and the number of personal trades{nl}will be accumulated and all other benefits will be maintained with the same values.
+ETC_20160323_021374	Guild Reward
+ETC_20160323_021375	1st Standing Server Reward
+ETC_20160323_021376	Free TP + {Value} until the next refresh of the weekly standings
+ETC_20160323_021377	Re-enter Automatic Party Matching
```
Comment (Sunaries) [inline EnglishTranslation-master/ETC.tsv:21377]: Auto

File: EnglishTranslation-master/ETC.tsv [hunk 10]
```diff
@@ -15153,7 +15153,7 @@ ETC_20151001_015154	I am counting on you!
 ETC_20151001_015155	I'm alive... Thank you!
 ETC_20151001_015156	You've failed to protect Hix
 ETC_20151001_015157	The shrine has restored to its previous status
-ETC_20151001_015158	You've succeeded activating Romuva magic field
+ETC_20151001_015158	You've succeeded in activating the Romuva magic circle
```
Comment (Sunaries) [inline EnglishTranslation-master/ETC.tsv:15156]: Capitalize magic circles (lowercase without names)

**Comment by Sunaries** [comment]
Note to myself: 

Attribute: umbo blow: add knockdown
(Desc says knockBACK power) <~ check

**Comment by Sunaries** [comment]
Note to self: check Jay's pm on Discord for the fix to a Featherfoot skill.

**Comment by Sunaries** [comment]
Note to self: check Paladin's conversion (enemy -> enemies)

**Comment by Sunaries** [comment]
Good to merge.

**Comment by Sunaries** [comment]
Good to merge.

**Comment by imcgames** [comment]
:+1:

**Pull request merged**